tests/Behat/MediaCreationContext.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content\Behat;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Drupal\Tests\oe_content\Traits\UtilityTrait;
+
+/**
+ * Defines media entity creation related step definitions.
+ */
+class MediaCreationContext extends RawDrupalContext {
+
+  use UtilityTrait;
+
+  /**
+   * Keep track of medias so they can be cleaned up.
+   *
+   * @var array
+   */
+  protected $medias = [];
+
+  /**
+   * Keep track of files so they can be cleaned up.
+   *
+   * @var array
+   */
+  protected $files = [];
+
+  /**
+   * Creates media documents with the specified file names.
+   *
+   * // phpcs:disable
+   * @Given the following documents:
+   * | name 1 | file 1 |
+   * | name 2 | file 1 |
+   * |   ...  |   ...  |
+   * // phpcs:enable
+   */
+  public function createMediaDocuments(TableNode $file_table): void {
+    // Retrieve the url table from the test scenario.
+    $files = $file_table->getRows();
+
+    foreach ($files as $file_properties) {
+      $file = file_save_data(file_get_contents(drupal_get_path('module', 'oe_content') . '/tests/fixtures/' . $file_properties[1]), 'public://' . $file_properties[1]);
```

Comment:
We have to use here the `files_path` property in the module's `behat.yml.dist`, and not the module's path, as this is supposed to be a reusable behat context.

You can access it by calling `$this->getMinkParameter('files_path')`:

```php
$file = file_save_data(file_get_contents($this->getMinkParameter('files_path') . $file_properties[1]), 'public://' . $file_properties[1]);
```